components/usage/pkg/db/billed_session_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package db_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/gitpod-io/gitpod/usage/pkg/db"
+	"github.com/gitpod-io/gitpod/usage/pkg/db/dbtest"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBilledSession_WriteRead(t *testing.T) {
+	conn := dbtest.ConnectForTests(t)
+	teamID := uuid.New().String()
+	teamAttributionID := db.NewTeamAttributionID(teamID)
+
+	instance := dbtest.NewWorkspaceInstance(t, db.WorkspaceInstance{
+		ID:                 uuid.New(),
+		UsageAttributionID: teamAttributionID,
+		CreationTime:       db.NewVarcharTime(time.Date(2022, 05, 1, 00, 00, 00, 00, time.UTC)),
+		StartedTime:        db.NewVarcharTime(time.Date(2022, 05, 1, 00, 00, 00, 00, time.UTC)),
+		StoppedTime:        db.NewVarcharTime(time.Date(2022, 06, 1, 1, 0, 0, 0, time.UTC)),
+	})
+
+	billedSession := &db.BilledSession{
+		InstanceID: instance.ID,
+		From:       db.NewVarcharTime(time.Date(2022, 8, 04, 12, 00, 00, 00, time.UTC)),
+		To:         db.NewVarcharTime(time.Date(2022, 9, 04, 12, 00, 00, 00, time.UTC)),
+		System:     "chargebee",
+		InvoiceID:  "some-invoice-ID",
+	}
+
+	tx := conn.Create(billedSession)
```

Comment:
You should also delete the created record at the end of the test. For other DB models, there are helpers (in `dbtest` package) that do this automatically by hooking into the `t *testing.T` when the test finishes.